holonote/tests/test_connectors.py

```diff
@@ -131,3 +131,40 @@ def test_add_three_rows_delete_one(self, database):
         database.add_row(**insertion3)
         database.delete_row(id2)
         pd.testing.assert_frame_equal(database.load_dataframe(), df)
+
+
+@pytest.mark.parametrize(
```

Comment:
```suggestion
@pytest.mark.parameterize(
```

Was this working? `parametrize` has a typo!